data/blog/libtailscale.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "libtailscale: how it's made"
+title: 'libtailscale: put Tailscale in your C program'
```

Comment:
libtailscale: embed Tailscale in C using Go

Reply:
works for me!